voice.go

```diff
@@ -360,6 +360,25 @@ func (v *VoiceConnection) wsListen(wsConn *websocket.Conn, close <-chan struct{}
 				v.wsConn = nil
 				v.Unlock()
 
+				// Wait for VOICE_SERVER_UPDATE.
+				// When the bot is moved by the user to another voice channel,
+				// VOICE_SERVER_UPDATE is received after the code 4014.
+				for i := 0; i < 5; i++ {
+					<-time.After(1 * time.Second)
```

Comment:
Is there any reason why it's 5x1 second in particular?

Reply:
The client is reconnected almost within 1 second, but I set the timeout to 5 seconds to give more leeway.